### PR TITLE
Add references to miniforge in Readme and update links to miniforge in documentation.

### PR DESCRIPTION
**Pull request by SylvainCorlay**
### Description

The main changes is the update of the link for miniforge, now pointing to the relevant part of the conda-forge website rather than the GitHub repository.

I am also adding a reference to miniforge in the Readme.

### CODE REVIEW

**Comment by jezdez** [comment]
@conda-bot check

**Comment by jezdez** [comment]
@conda-bot check

**Comment by SylvainCorlay** [comment]
Ping @jezdez @beckermr. I think this is a first step on the discussion on advertising other installers not bound to channels that incur corsts.

File: README.md [hunk 1]
```diff
@@ -31,10 +31,9 @@ the amazing members of the conda community. Some of them can be found
 
 ## Installation
 
-Conda is a part of the [Anaconda Distribution](https://repo.anaconda.com).
-Use [Miniconda](https://docs.anaconda.com/free/miniconda/) to bootstrap a minimal installation
-that only includes conda and its dependencies.
+To bootstrap a minimal distribution, use a minimal installer such as [Miniconda](https://docs.anaconda.com/free/miniconda/) or [Miniforge](https://conda-forge.org/download/).
 
+Conda is also included  in the [Anaconda Distribution](https://repo.anaconda.com).
```
Comment (beckermr) [inline README.md:36]:
```suggestion
Conda is also included in the [Anaconda Distribution](https://repo.anaconda.com).
```

**Review by beckermr — APPROVED** [review]
One minor nit on spacing. LGTM!

**Comment by SylvainCorlay** [comment]
> One minor nit on spacing.

Fixed. Thanks!

**Review by jezdez — APPROVED** [review]
Thanks for the pull request, this is a good start!

**Pull request merged**